### ISSUE TITLE
Edge case with trailing backslashes

### ISSUE THREAD

**Issue by JohannesMP** [20Tauri/DoxyDoxygen#35]
Given these settings: https://gist.github.com/JohannesMP/51187060cb212c6c024d

This `preferred_comments_styles` currently works as expected:

```
[
  "/*!******************************************************************",
  "",
  "********************************************************************/"
],
```

![screenshot_2016-01-27_08 11 37](https://cloud.githubusercontent.com/assets/662874/12619630/ad965ef2-c4cd-11e5-8135-dcf1332fbed7.png)

However if I modify it to have leading and trailing backslashes as follows (which have to be escaped to be valid JSON):

```
[
  "/*!*****************************************************************\\",
  "",
  "\\*******************************************************************/"
],
```

The style seems to be ignored completely, reverting back to the default:

![screenshot_2016-01-27_08 12 00](https://cloud.githubusercontent.com/assets/662874/12619634/b19695ee-c4cd-11e5-85a3-1e2facf06b5f.png)

Testing seems to indicate that this is due to the trailing `\\` at the end of the first line.

---

Instead of reverting to the default, this is the expected output:

![screenshot_2016-01-27_08 13 22](https://cloud.githubusercontent.com/assets/662874/12619683/f46f75ac-c4cd-11e5-8ec5-db791370c58c.png)

As far as I've been able to test with Doxygen 1.8.10 this is a valid Doxygen header. DoxyDoxygen should be able to generate it.

**Comment by 20Tauri** [comment]
Thanks you,

I have reproduce it.
There's an error in console: "invalid configuration" (even if configuration IS VALID)

This have been fixed in 0.32.0

**Comment by JohannesMP** [comment]
Thanks for the fast fix! I can confirm that it now works for me.
